src/web/transactions.rs

```diff
@@ -151,7 +136,9 @@ async fn show(
     let hash = hash.into_inner().0;
     let tx = app
         .iroha_client
-        .request(QueryBuilder::new(FindTransactionByHash::new(hash)))
+        .request(QueryBuilder::new(FindTransactionByHash::new(
+            HashOf::from_untyped_unchecked(hash),
+        ))) // deprecated
```

Comment:
If you really need to leave a useful comment, extend it and explain, what is deprecated and why should we care about it.